hw/ip/prim_generic/lint/prim_generic_ram_1p.waiver

```diff
@@ -4,8 +4,6 @@
 #
 # waiver file for prim_generic_ram_[1,2]p
 
-waive -rules MULTI_PROC_ASSIGN -location {prim_generic_ram_2p.sv} -regexp {Assignment to 'storage' from more than one block} \
-      -comment "That is the nature of a dual-port memory: both write ports can access the same storage simultaneously"
```

Comment:
was anything done specifically for this..?

Reply:
so the thing is that this is not needed for the single-port RAM. if you look at the `location` of that rule, this is inside the dual port RAM implementation - which is wrong. the linter complains that it can't find `prim_generic_ram_2p.sv` in builds where the dual port implementation is not included.